ui/components/multichain/pages/send/components/your-accounts.tsx

```diff
@@ -25,7 +27,12 @@ export const SendPageYourAccounts = () => {
   // Your Accounts
   const accounts = useSelector(getUpdatedAndSortedAccounts);
   const internalAccounts = useSelector(getInternalAccounts);
-  const mergedAccounts = mergeAccounts(accounts, internalAccounts);
+  const mergedAccounts: InternalAccount &
+    { keyring: KeyringTypes; label: string }[] = useMemo(() => {
```

Comment:
You will also need to import `MergedInternalAccount`

```suggestion
  const mergedAccounts: MergedInternalAccount[] = useMemo(() => {
```